# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chromium/qt3.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./../Chromium-0.9/src-setup/HelpPage.cpp	Thu Feb  8 06:18:59 2001
+++ ./src-setup/HelpPage.cpp	Thu Apr  4 20:56:02 2002
@@ -22,8 +22,8 @@
 	cgroup.setColor(QColorGroup::Base, QColor(70, 60, 100));
 	cgroup.setColor(QColorGroup::Text, QColor(220, 220, 220));
 	QFont	font("Helvetica", 12, false);
-	page->setPaperColorGroup(cgroup);
-	page->setLinkColor(QColor(192, 192, 255));
+	//page->setPaperColorGroup(cgroup);
+	//page->setLinkColor(QColor(192, 192, 255));
 	page->setFont(font);
 
 	backBut = new QPushButton(this);
